src/wallet/rpcdump.cpp

```diff
@@ -989,8 +990,10 @@ UniValue z_exportviewingkey(const UniValue& params, bool fHelp)
     if (fHelp || params.size() != 1)
         throw runtime_error(
             "z_exportviewingkey \"zaddr\"\n"
-            "\nReveals the viewing key corresponding to 'zaddr'.\n"
-            "Then the z_importviewingkey can be used with this output\n"
+            "\nReturns the full viewing key corresponding to 'zaddr' in the format specified by\n"
+            "https://zips.z.cash/protocol/protocol.pdf#addressandkeyencoding.\n"
+            "z_importviewingkey can be used to import Sprout and Sapling exported in this format;\n"
+            "import of unified viewing keys is not yet supported.\n"
```

Comment:
For consistency, should this be "Unified viewing..." to be consistent with line 825 for example?